setup.cfg

```diff
@@ -1,10 +1,3 @@
 [flake8]
 ignore = E302,W503
 max-line-length = 119
-
-[isort]
-line_length = 119
-multi_line_output = 5
```

Comment:
This changes multi_line_output from 5 to 3, and removes the line-length setting from isort. Is that intentional?

Reply:
Yes.
1. To resolve conflicts between isort and black [multi_line_output should be set to 3](https://pycqa.github.io/isort/docs/configuration/black_compatibility.html).
2. Currently, black uses standard pep-8 line_length=79. We should sync the line_length in all of our formatting tools (isort/black/flake8). We can set all of them to 119 if you prefer.
3. The main reason for this PR was cleaning duplicated configs in two files.

Reply:
Removing duplication is great. 

I think we should have the same line-length everywhere. I prefer 119, but it isn’t that important.